services/app-web/src/contexts/AuthContext.tsx

```diff
@@ -188,17 +187,15 @@ function AuthProvider({
     /*
         Refreshes the cognito session token
         Also can reconfirm authentication, if unexpectedly logged out we know that session may have timed out
-        @param {failureRedirectPath} passed through to logout which is called on certain checkAuth failures
     */
     const refreshAuth = async () => {
         if (authMode !== 'LOCAL') {
             const result = await extendSession()
-            if (result instanceof Error) {
-                if (loggedInUser) {
-                    await logout({
-                        type: 'TIMEOUT',
-                    })
-                }
+            if (result instanceof Error){
+                await logout({
+                    type: 'TIMEOUT'
+                })
+
```

Comment:
```suggestion
```

Somethings up with there are extra spaces and new lines not being caught by `prettier` or being fixed by pre-commit.